health/notifications/email/README.md

```diff
@@ -30,4 +30,26 @@ sudo su -s /bin/bash netdata
 
 Where `[ROLE]` is the role you want to test. The default (if you don't give a `[ROLE]`) is `sysadmin`.
 
+# Simple SMTP transport configuration
+
+If you want an alternative of sendmail in order to have a simple MTA configuration for sending emails and auth to an existing SMTP server :
+
+- Remove all sendmail stuff.
+- Install msmtp.
+- Make a symbolic link of /usr/bin/msmtp to /usr/sbin/sendmail (default debian location of sendmail) :
```

Comment:
- Make `/usr/sbin/sendmail` (default debian location of `sendmail`) a symbolic link to `/usr/bin/msmtp` :

However, you don't need this for netdata actually. You can instead have the following:

- Modify the `sendmail` path in `health_alarm_notify.conf` to point to the location of `mstmp`:
```
# The full path to the sendmail command.
# If empty, the system $PATH will be searched for it.
# If not found, email notifications will be disabled (silently).
sendmail="/usr/bin/msmtp"
```

Reply:
oh god ! exactly ! thats the fastest solution

Reply:
The main advantage of mapping msmtp to sendmail is : your msmtp configuration is global for your system and the softwares. like unified lol. (do that if you only need server push to a mailbox).